setup.py

```diff
@@ -480,7 +480,7 @@ def get_file_list (self):
     },
     # Requirements, usable with setuptools or the new Python packaging module.
     install_requires = [
-        'requests >= 2.2.0',
+        'requests<2.15,>=2.2',
```

Comment:
oh. i see. somehow I thought 2.15 was *lower* than 2.2.0 :p everything is fine then. i would suggest we keep #74 open to fix 2.15+ support properly, however.